modules/auth/types.go

```diff
@@ -0,0 +1,31 @@
+package auth
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// --- Types are defined according to genesis file structure ---
+
+type Accounts struct {
```

Comment:
Why are all these types defined here? What is their utility? Can't we simply use the Cosmos ones?